tests/unit/test_enrichment_router.py

```
@@ -11,22 +11,68 @@
 from api.routers.enrichment import enrichment_router
 from internal.middleware.fastapi_auth import AuthContext
 from db.connection import get_db
-from pkg.errors.app_exceptions import AppException, ValidationException
-from tests.unit.conftest import make_mock_session
+from pkg.errors.app_exceptions import AppException, NotFoundException, ValidationException
+from pydantic import ValidationError
+from tests.unit.conftest import make_mock_session, make_customer_handler, MockState
```

Pylint (4.0.5)

[error] No name 'make_customer_handler' in module 'tests.unit.conftest'

(E0611)